README.md

```diff
@@ -138,12 +145,109 @@ make test-ts-binary-rococo
 
 Remember to run the clean-up afterwards.
 
-## extend the leasing period
+## License
 
-The default leasing duration for parachain is 1 day, in case you want to extend it (even after it's downgraded to parathread), simply do a `forceLease` via sudo, it should be upgraded to parachain soon again and start to produce blocks.
+GPLv3
 
-![image](https://user-images.githubusercontent.com/7630809/135689832-1f57cd5c-7f83-4fce-9bb0-832b77a38dcc.png)
 
-## License
+## How to Build and Run Parachain and Tee-worker
 
-GPLv3
+## Preparation
+
+1. env: [Setup **SGX TEE** Environment](https://web3builders.notion.site/Setup-SGX-TEE-Environment-68066770831b45b7b632e682cf159477?pvs=4) 
+2. add VSCode configs for indexing and parsing tee-worker codes
+
+```
+"rust-analyzer.linkedProjects": [
+        "tee-worker/Cargo.toml"
+    ],
+```
+
+## Build
+
+We need to build the `integritee-service` before we launch the entire system
```

Comment:
Here, it's not `integritee-service` any more. Renamed to: `litentry-service`.

Reply:
it's actually `litentry-worker`